core/common/src/main/java/alluxio/network/TieredIdentityFactory.java

```diff
@@ -78,6 +80,11 @@ static TieredIdentity create() {
       }
       tiers.add(new LocalityTier(tierName, value));
     }
+    if (tiers.size() > 0 && tiers.get(0).getTierName().equals(Constants.LOCALITY_NODE)
+        && tiers.get(0).getValue() == null) {
```

Comment:
are the other tier values going to be not-null? Does it make sense if a value is `null`?

Reply:
A value of `null` means that the tier is defined in PropertyKey.LOCALITY_ORDER, but the user didn't specify a value for it. If the user is working from default configuration, the `node` and `rack` tiers will have `null` values. We leave `rack` as `null` since we don't have a good guess for what rack they want, but use the local node name as a default value for `node`.